src/__main__.py

```diff
@@ -30,6 +30,7 @@
     "demiboy": [244, 249, 117, 255, 117, 249, 244],
     "demigirl": [244, 249, 218, 255, 218, 249, 244],
     "genderfluid": [211, 255, 128, 0, 63],
+    "genderqueer": [141, 141, 255, 255, 64, 64],
```

Comment:
Pridefetch automatically scales flags, so the duplication of each colour is not required here. This is seen in the pansexual flag. The only reason a colour would be duplicated is to make it a bigger part of a flag, such as the lesbian flag. Instead - use `"genderqueer": [141, 255, 64]`.

Reply:
```suggestion
    "genderqueer": [141, 255, 64],
```